site-content/source/modules/ROOT/pages/blog/Inside-Cassandra-an-interview-with-Project-Contributor-Lorina-Poland.adoc

```diff
@@ -0,0 +1,38 @@
+= Inside Cassandra: An Interview with Project Contributor, Lorina Poland
+:page-layout: single-post
+:page-role: blog-post
+:page-post-date: March 17, 2022
+:page-post-author: The Apache Cassandra Community
+:description: The Apache Cassandra Community
+:keywords: 
+
+It’s time to start putting some names and faces to all the work that goes into the Apache Cassandra project. The community powers everything we do. Without them, we wouldn’t be building on such a stable release, seeing so many features being merged and in development, and you wouldn’t be able to read this website right now!
+
+=== About Our Contributor
+
+image:blog/inside-Cassandra-Lorina-Poland.jpg[Lorina Poland,float="left"] Our first contributor interview is with Lorina Poland, who lives in California. Lorina’s first job out of college was writing programs in Fortran for Chevron R&D that we used in fieldwork in oilfields. She went on to become an Engineering Officer in the U.S. Air Force until retiring in 2007. Because military service wasn’t demanding enough, she spent ten years teaching high school kids math and science. In her current role, she works full-time as a lead technical writer for DataStax.
+
+As a committer for Apache Cassandra, she tends to focus on documentation. Her main professional interests are better maintaining and publishing tools for documentation, and recently, APIs for interacting with Apache Cassandra.
+
+=== Contributor Questions
+
+*Question:* What are you currently working on, or have worked on, in the past for the Apache Cassandra project?
```

Comment:
I'm adding a blank line after each question, otherwise the "answers" don't render correctly:

![bad rendering](https://issues.apache.org/jira/secure/attachment/13041254/c17441-01-bad_render.png)

```suggestion
*Question:* What are you currently working on, or have worked on, in the past for the Apache Cassandra project?

```